### PR TITLE
make the 1.0.0 stable release, via rubygems

### PR DESCRIPTION
**Pull request by diclophis**
@cantino / @giovannibonetti this is pushed up here: https://rubygems.org/gems/alaska

### CODE REVIEW

**Comment by diclophis** [comment]
I had to bump to `1.0.1` to fix small issue with test suite on travis

**Comment by cantino** [comment]
Nice!

File: alaska.gemspec [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-  s.version     = '0.0.8'
+  s.version     = '1.0.0'
   s.date        = '2015-02-10'
-  s.summary     = "caching ExecJS runtime"
-  s.description = ""
-  s.authors     = ["Jon Bardin"]
+  s.summary     = "persistent ExecJS runtime"
+  s.description = "uses a single shared nodejs process to handle ExecJS::Runtime evaluation"
+  s.authors     = ["Jon Bardin", "Stephen Grider", "Ville Lautanala", "Giovanni Bonetti"]
   s.email       = 'diclophis@gmail.com'
```
Comment (cantino) [inline alaska.gemspec]: You can change this to opensource@mavenlink.com if you don't want to get spammed.

**Comment by diclophis** [comment]
I think this failed randomly... https://travis-ci.org/mavenlink/alaska/jobs/91481164

**Pull request merged**